docarray/typing/tensor/tensorflow_tensor.py

```diff
@@ -0,0 +1,141 @@
+from typing import TYPE_CHECKING, Any, Dict, Generic, Type, TypeVar, Union
+
+import numpy as np
+import tensorflow as tf
+
+from docarray.typing.proto_register import _register_proto
+from docarray.typing.tensor.abstract_tensor import AbstractTensor
+
+if TYPE_CHECKING:
+    from pydantic.fields import ModelField
+    from pydantic import BaseConfig
+    from docarray.proto import NdArrayProto
+    from docarray.computation.tensorflow_backend import TensorFlowCompBackend
+
+from docarray.base_document.base_node import BaseNode
+
+T = TypeVar('T', bound='TensorFlowTensor')
+ShapeT = TypeVar('ShapeT')
+
+tf_base: type = type(tf.Tensor)
+node_base: type = type(BaseNode)
+
+
+class metaTensorFlow(
+    AbstractTensor.__parametrized_meta__,  # type: ignore
+    node_base,  # type: ignore
+    tf_base,  # type: ignore
+):  # type: ignore
+    pass
+
+
+@_register_proto(proto_type_name='tensorflow_tensor')
+class TensorFlowTensor(AbstractTensor, Generic[ShapeT], metaclass=metaTensorFlow):
+
+    __parametrized_meta__ = metaTensorFlow
+
+    def __init__(self, tensor: tf.Tensor):
+        super().__init__()
+        self._tensor = tensor
+
+    @property
+    def tensor(self):
+        return self._tensor
```

Comment:
Since when do we have a `tensor` property and an `unwrap()` method that do the same thing?

Reply:
I would leave it public and just keep the `unwrap()` so that the interface is uniform between tf and the other ones (and the other ones don't have a `tensor` that they store).
`unwrap()` seems to be in the abstract tensor already, which is all we need i think :)